packages/nc-gui-v2/components/cell/MultiSelect.vue

```diff
@@ -67,19 +96,32 @@ watch(
     })
   },
 )
+
+watch(
+  () => isOpen.value,
+  (n, _o) => {
+    if (n === false) {
+      aselect.value.blur()
+    }
+  },
+)
```

Comment:
Watch doesn't need a getter function if you're going to watch a `ref` or `computed` value.
```ts
watch(isOpen, () => { ... })
```